source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -1163,12 +1163,19 @@ def isPointInObject(self, x, y):
 		return True
 
 	def _get_labeledBy(self):
```

Comment:
please add a return type

```suggestion
	def _get_labeledBy(self) -> IAccessible | None:
```